crates/derive/src/batch/validity.rs

```diff
@@ -15,6 +15,9 @@ pub enum BatchValidity {
     Undecided,
     /// The batch may be valid, but cannot be processed yet and should be checked again later
     Future,
+    /// Introduced in Holocene, a special variant of the Drop variant that signals not to flush
+    /// the active batch and channel.
+    BatchOutdated,
```

Comment:
Following the others enum names, wouldn't you just name this `Outdated`? Note that I name the validity `past` in the specs. Or do you strongly prefer `outdated` over `past`? I thought that `past` pairs better with `future`.

Reply:
Not opinionated. We can standardized on the `Past` name.

Reply:
Ok, also using `past` in the Go implementation now.